test/compile_errors.zig

```diff
@@ -1,6 +1,24 @@
 const tests = @import("tests.zig");
 
 pub fn addCases(cases: *tests.CompileErrorContext) void {
+    cases.add(
```

Comment:
Pretty sure this tests compiler errors for compiler features, so these should not be here anymore.

Reply:
`std.math.minValue` and friends emits a `@compileError` for using types that are not an Integer -- this was supposed to test that, but where would be the better place?

Reply:
I don't think `std` tests for compiler errors for any of its functions. We probably just have to wait for some feature that will allow for this (I feel like I've seen an issue for it somewhere, but have only found #1356 and #567)